config/routes.php

```diff
@@ -20,4 +20,15 @@
         '/panels/*',
         ['controller' => 'Panels', 'action' => 'index']
     );
+
+    $routes->scope(
+        '/mail_preview',
```

Comment:
The thing is that the plugin url is `debug_kit`, so it looks a bit weird to mix styles